ui/src/containers/Visualise/CopyButton.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useCallback } from 'react';
+import { connect } from 'react-redux';
+import { compose, withHandlers, mapProps } from 'recompose';
+import { withModel } from 'ui/utils/hocs';
+import ConfirmModal from 'ui/components/Modal/ConfirmModal';
+import { loggedInUserId } from 'ui/redux/modules/auth';
+import { COPY_VISUALISATION } from 'ui/redux/modules/visualisation/copyVisualisation';
+
+/**
+ * @param {function} _.copyVisualisation
+ */
+const CopyIconButton = ({ copyVisualisation }) => {
```

Comment:
Do you think we should extract the copy button out in the dashboard component like we've done here for the visualisations?

Reply:
ok